clients/osv/src/funTest/kotlin/OsvServiceFunTest.kt

```diff
@@ -50,14 +48,12 @@ private val VULNERABILITY_FOR_PACKAGE_BY_INVALID_COMMIT_REQUEST = Vulnerabilitie
 private fun Vulnerability.patchIgnorableFields() =
     copy(
         modified = Instant.EPOCH,
-        databaseSpecific = emptyJsonObject.takeIf { databaseSpecific != null },
+        databaseSpecific = null,
```

Comment:
commit: Why is the old format continued to be supported? Does OSV now return a mixture of both formats?

Reply:
Looking at the expected result, it seems that only the new format is returned. So, can we adjust this PR to handle the new format only, thereby eliminating the unneeded JsonNames annotations again?

Reply:
> So, can we adjust this PR to handle the new format only, thereby eliminating the unneeded JsonNames annotations again?

I'm currently testing to do this. Initially, it seemed to me as if the old format is still returned under some circumstances, and also @mnonnenmacher reported something in that direction to me, but at least in my case the cause was likely outdated test data.